### ISSUE TITLE
Migrating from v292 FlyByWire import crashing Blender 4.2

### ISSUE THREAD

**Issue by rhumbaflappy** [ronh991/glTF-Blender-IO-MSFS#16]
### Current Behavior

Opening a v2.92 .blend file from the FlyByWire MSFS importer works OK. Pre-migrating the materials doesn't get the comp file added. Then Migrating the material crashes Blender. Using Blender 4.2 LTS and your 4.2 MSFS extension.

### Expected Behavior

No crashing

### Steps To Reproduce

As above
[https://drive.google.com/file/d/1F1OgaD6sU6QynL_rgG1MLR5DsfubvFDG/view?usp=sharing](url)

### Environment

```markdown
- OS:Windows 11
- Blender:4.2 LTS
- glTF-Blender-IO-MSFS:4.2 Beta
```

**Comment by rhumbaflappy** [comment]
Also the recent 3.6 also has a migration problem in that the base color and emissive color are getting messed up. Sometimes going to black, sometimes going to 50% alpha, sometimes going with a green hue.

The Asobo for 3.6 seem to be OK.

**Comment by ronh991** [comment]
investigating this issue

**Comment by ronh991** [comment]
As for the FBW importer the comp texture for detail and regular an being populated to the msfs_comp_texture variable. So my export will not see those textures and place them into the msfs_occlusion_metal_roughness variable.

See FBW gltf2_blender_texture.py lines 72 to 96

**Comment by ronh991** [comment]
see if this fixes your issue.  if not please provide a blend file that shows the problem.
[io_scene_gltf2_msfs_for42_ext.zip](https://github.com/user-attachments/files/16457435/io_scene_gltf2_msfs_for42_ext.zip)

**Comment by ronh991** [comment]
in my testing I found that the FBW importer is not doing detail_texture data scale.

adding 
```

            if "scale" in mh.pymat.extensions["ASOBO_material_detail_map"]["detailNormalTexture"]:
                print("check normal scale found")
                scale = mh.pymat.extensions["ASOBO_material_detail_map"]["detailNormalTexture"][
                        "scale"
                    ]
                mh.mat.msfs_detail_normal_scale = scale
                print("check normal scale set", scale)
                create_detail_UV = True
            # detailNormalTexture scale

```

to gltf2_blender_pbrMetallicRoughness.py at line 108 in FBW importer will help - could probably add for other detail textures.

**Comment by rhumbaflappy** [comment]
The addon zip seems to work. The only thing I noticed is that the Migrate function loses the base emissive color. The .blend file I used has a parallax material, and the emissive color goes to black. But no crashing now!

One other issue I see is the Vertex Color Scale defaults to 0.5 and it might be better to default to 1.0 as it is a multiplier factor.

**Comment by ronh991** [comment]
Can you supply a blend fil with the parallax material I can test with, or point me to an object I can import.

**Comment by ronh991** [comment]
may have found the emissive bug , and I changed default vertex.  Please test if you can
[io_scene_gltf2_msfs_for42_ext.zip](https://github.com/user-attachments/files/16476919/io_scene_gltf2_msfs_for42_ext.zip)

**Comment by rhumbaflappy** [comment]
It works. The FBW import for 2.92 might incorrectly make the Parallax as Detail? Once I fixed that, all was good with your new 4.2.

**Comment by ronh991** [comment]
Great, thanks for the bug report.
